src/components/accession2/create/Accession2Create.tsx

```diff
@@ -45,6 +49,21 @@ export default function CreateAccession(): JSX.Element {
   const snackbar = useSnackbar();
   const [validateFields, setValidateFields] = useState<boolean>(false);
   const [record, setRecord, onChange] = useForm<AccessionPostRequestBody>(defaultAccession());
+  const [selectedSeedBank, setSelectedSeedBank] = useState<Facility>();
```

Comment:
on line # 38 we have to replace the default receivedDate date to use the time zone, we have to pass the timezone into the function maybe.. some logic we have add.
There's a new util in web-components/utils/date.ts getTodaysDateFormatted(timeZone) we can use that